### PR TITLE
test(e2e): mint handles on localnet start

### PR DESCRIPTION
**Pull request by iadmytro**
# Context

Minting Ada handle on local test network

### CODE REVIEW

**Review by iccicci — APPROVED** [review]
LGTM

File: packages/e2e/local-network/scripts/mint-tokens.sh [hunk 1]
```diff
@@ -47,7 +47,7 @@ done
 
 cardano-cli transaction build \
   --babbage-era \
-  --change-address "$faucetAddr" \
+  --change-address "$addr" \
```
Comment (rhyslbw) [inline packages/e2e/local-network/scripts/mint-tokens.sh:50]: Bug? Next time it's best to commit separately, still as a `test` type, but it's part of a different change.

File: packages/e2e/local-network/README.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ Supported OS:
 1. Run `./scripts/install.sh` to install necessary binaries.
 2. Run `./scripts/make-babbage.sh` to create all the keys, certificates and genesis files.
 3. Run `./scrips/start.sh` to start the nodes.
-4. Run `export CARDANO_NODE_SOCKET_PATH=$PWD/network-files/node-spo1/node.sock` for `cardano-cli` to work.
+4. Run `export CARDANO_NODE_SOCKET_PATH=$PWD/network-files/node-sp1/node.sock` for `cardano-cli` to work.
```
Comment (rhyslbw) [inline packages/e2e/local-network/README.md:24]: Also could be a separate `docs` commit

**Review by rhyslbw — APPROVED** [review]
Nice work @iadmytro. The feedback is for future reference.

File: packages/e2e/local-network/scripts/mint-tokens.sh (hunk 1 above)
Comment (iadmytro) [reply packages/e2e/local-network/scripts/mint-tokens.sh:50]: The funds should stay in address for next script (mint-handle) to use them.

Comment (rhyslbw) [reply packages/e2e/local-network/scripts/mint-tokens.sh:50]: Oh, so functional change to support this extension. Got it

**Review by iccicci — APPROVED** [review]

**Review by rhyslbw — APPROVED** [review]

**Pull request merged**